.npmrc

```diff
@@ -4,9 +4,12 @@
 # it might get the wrong version of.
 hoist = false
 
-# We _do_ try to hoist `@types` packages so that TS will not end up with type
-# errors from comparing types from different versions of the same package.
-public-hoist-pattern=['@types/*']
```

Comment:
I think due to incorrect format, this `public-hoist-pattern` was not doing anything.